project/Build.scala

```diff
@@ -155,8 +155,10 @@ object AlgebirdBuild extends Build {
   )
 
   lazy val algebirdTest = module("test").settings(
+    testOptions in Test ++= Seq(Tests.Argument(TestFrameworks.ScalaCheck, "-verbosity", "4"),
+                                Tests.Argument(TestFrameworks.ScalaCheck, "-minSuccessfulTests", "1")),
```

Comment:
https://github.com/rickynils/scalacheck/issues/187
Looks like the Scalacheck guy might add this feature in Scalacheck 2.0.0 (in a long time)

Any ideas for a workaround?